OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -52,147 +52,604 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        </explicit>
+                                        <note>The <foreign xml:lang="la">tituli</foreign> are numbered from 1 to 19.</note>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.4.2">
+                                        <locus from="101ra" to="123vb"/>
+                                        <title type="incomplete" ref="LIT2715John"/>
+                                        <textLang mainLang="gez"/>
+                                        <incipit xml:lang="gez" type="inscription">
+                                            <locus target="#101r"/>
+                                            <hi rend="rubric">ወንጌል፡ ዘዮሐንስ፡</hi>
+                                        </incipit>
+                                        <incipit xml:lang="gez">
+                                            <locus target="#101ra"/>
+                                            <hi rend="rubric">ቀዳሚሁ፡ ቃል፡ ውእቱ። ወውእ</hi>ቱ፡ ቃል፡ ኃበ፡ እግዚአ፡ ብሔር፡ <hi rend="rubric">ውእቱ፡ ወእግዚአ፡ ብሔር፡ ውእ</hi>ቱ፡ 
+                                            ቃል። ወከማሁ፡ ቀዳሚሁ፡ እ<hi rend="rubric">ምቅዲሙ፡ ኃበ፡ እግዚአ፡ ብሔር፡</hi> ውእቱ። ወኵሉ፡ ቦቱ፡ ኮነ፡ ወዘእ<hi rend="rubric">ንበሌሁሰ፡ 
+                                                አልቦ፡ ዘኮነ፡ ወኢምን</hi>ትኒ። ወዘሂ፡ ኮነ፡ በእንቲኣሁ፡ ቦቱ፡ <hi rend="rubric">ሕይወት፡ ውእቱ። ወሕይወትሰ፡</hi> ብርሃኑ፡ ለእጓለ፡ እመ፡ ሕያው፡ 
+                                            ውእቱ።
+                                        </incipit>
+                                        <explicit xml:lang="gez">
+                                            <locus target="#123vb"/>
+                                            ኵሎ፡ ዘቦ፡ አቡየ፡ ዚአየ፡ ውእቱ። በእንተ፡ ዝንቱ፡ እቤ፡ ለክሙ፡ እምዚአየ፡ ይነሥእ፡ ወይነግረክሙ። ሕዳጠ፡ ወኢትሬእዩኒ፡ ወካዕበ፡ ሕዳጠ፡ ወትሬእዩኒ። 
+                                            ወቦ፡ እለ፡ ይቤሉ፡ እምኣርዳኢሁ፡ በበይናቲሆሙ፡ ምንተኑ፡ ዘይብለነ፡ ሕዳጠ፡ ወኢትሬእዩኒ፡ ወካዕበ፡ ሕዳጠ፡ ወትሬእዩኒ፡ ወአሓወር፡ ኃበ፡ ኣብ። ወይቤሉ፡ 
+                                            ምንትኑ፡ ው
+                                        </explicit>
+                                        <note>The text terminates abruptly at Joh. 16:18.</note>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+                        </msContents>
+                        
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                        <p>Holes, presumably resulting from parchment making, are found on, e.g., <locus target="#1 #40 #85"/>.
+                                            Tears on several leaves, e.g., on <locus target="#2 #106 #118"/> have been repaired at an undetermined point in time.</p>
```

Comment:
@SusanneHummel are the commas around e.g. correct?

Reply:
Comma before, not after: 



![IMG_20201026_175134](https://user-images.githubusercontent.com/28757489/97202887-7b975700-17b4-11eb-88b5-e381bff47c19.jpg)